common/test/common/prices/PriceFinderTest.java

```diff
@@ -0,0 +1,229 @@
+package common.prices;
+
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.customergroups.CustomerGroup;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.products.Price;
+import org.javamoney.moneta.CurrencyUnitBuilder;
+import org.junit.Test;
+
+import javax.money.CurrencyContext;
+import javax.money.CurrencyContextBuilder;
+import javax.money.CurrencyUnit;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Optional;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PriceFinderTest {
+
+    private final PriceFinder priceFinder = PriceFinder.of();
+
+    private final CurrencyContext eurContext = CurrencyContextBuilder.of("EUR").build();
```

Comment:
Can you please use a more idiomatic way to initialize currencies?

`Monetary.getCurrency("EUR")`

I think currency context is not needed. Could you replace it, in the whole project?

http://sphereio.github.io/sphere-jvm-sdk/javadoc/master/io/sphere/sdk/meta/FormattingDocumentation.html#creating-money-values